dev/docker-compose.yaml

```diff
@@ -276,6 +276,24 @@ services:
     depends_on:
       - redis
 
+  isaccml:
+    image: ghcr.io/uwcirg/isacc-ml:${ISACCML_IMAGE_TAG:-main}
+    environment:
+      PREFERRED_URL_SCHEME: https
```

Comment:
I think this was a config needed only for sending emails (where there is no HTTP context to infer HTTP/S from)
```suggestion
```